style.css

```diff
@@ -0,0 +1,3 @@
+.head{
+	color: blue;
+}
```

Comment:
I think it would be better to enter an empty line and the end of the CSS file, doesn't it?